test/plexy/instrumentor_test.exs

```diff
@@ -15,35 +15,28 @@ defmodule Plexy.InstrumentorTest do
   end
 
   @opts Instrumentor.init([])
-  test "Instrumentor.call/2 logs the start" do
-    logged =
-      capture_log(fn ->
-        conn(:get, "/foobar") |> Instrumentor.call(@opts)
-      end)
-
-    [start_log_line | _finish_log_line] = String.split(logged, "\n")
-
-    assert start_log_line =~ "instrumentation"
-    assert start_log_line =~ "start"
-    assert start_log_line =~ "path"
-    assert start_log_line =~ "method"
-  end
-
-  test "Instrumentor.call/2 logs the finish" do
+  test "Instrumentor.call/2 logging" do
     logged =
       capture_log(fn ->
         conn(:get, "/foobar")
         |> Instrumentor.call(@opts)
         |> Plug.Conn.send_resp(200, "wow")
       end)
 
-    [_start_log_line, finish_log_line | _empty_list] = String.split(logged, "\n")
+    [start_log_line, measure_log_line, finish_log_line | _empty_list] = String.split(logged, "\n")
+
+    assert start_log_line =~ "instrumentation=true"
```

Comment:
I really like these changes as it makes it much easier to understand what to expect in the logline